test/support/builders/atom_watcher_event.js

```diff
@@ -0,0 +1,70 @@
+const metadata = require('../../../core/metadata')
+const events = require('../../../core/local/steps/event')
+
+/*::
+import type { Stats } from 'fs'
+import type { AtomWatcherEvent, Action, Kind, Batch } from '../../../core/local/steps/event'
+*/
+
+function randomPick (elements /*: Array<T> */) /*: T */{
+  const l = elements.length
+  const i = Math.floor(Math.random() * Math.floor(l))
```

Comment:
Why `Math.floor(l)`?

Reply:
To have an integer between `0` and `elements.length`

Reply:
Ah I see what you mean. `l` is already an integer. I'll fix that.